proto3-suite.cabal

```diff
@@ -103,6 +103,7 @@ library
   build-depends:       aeson >= 1.1.1.0 && < 2.2,
                        aeson-pretty,
                        attoparsec >= 0.13.0.1,
+                       attoparsec-aeson,
```

Comment:
What is `attoparsec-aeson` used for? If we absolutely must depend on `attoparsec-aeson` then can we have bounds added?